.github/labels.yml

```diff
@@ -0,0 +1,324 @@
+  color: *collectors
+- name: collectors/charts.d
+  description: Issues and PRs related to the charts.d data collection framework
+  color: *collectors
+- name: collectors/cups
+  description: Issues and PRs related to the CUPS data collection plugin
+  color: *collectors
+- name: collectors/diskspace
+  description: Issues and PRs related to the diskspace data collection plugin
+  color: *collectors
+- name: collectors/ebpf
+  description: Issues and PRs related to the eBPF data collection plugin
+  color: *collectors
+- name: collectors/freebsd
+  description: Issues and PRs related to the FreeBSD data collection plugin
+  color: *collectors
+- name: collectors/freeipmi
+  description: Issues and PRs related to the FreeIPMI data collection plugin
+  color: *collectors
+- name: collectors/go.d
+  description: Issues and PRs related to the go.d data collection framework
+  color: *collectors
+- name: collectors/idlejitter
+  description: Issues and PRs related to the idlejitter data collection plugin
+  color: *collectors
+- name: collectors/ioping
+  description: Issues and PRs related to the ioping data collection plugin
+  color: *collectors
+- name: collectors/macos
+  description: Issues and PRs related to the macOS data collection plugin
+  color: *collectors
+- name: collectors/nfacct
+  description: Issues and PRs related to the NFACCT data collection plugin
+  color: *collectors
+- name: collectors/new
+  description: Issues and PRs related to adding a new data collection plugin
+  color: *collectors
+- name: collectors/perf
+  description: Issues and PRs related to the Linux perf data collection plugin
+  color: *collectors
+- name: collectors/plugins.d
+  description: Issues and PRs related to the plugins.d data collection framework
+  color: *collectors
+- name: collectors/proc
+  description: Issues and PRs related to the proc data collection plugin
+  color: *collectors
+- name: collectors/python.d
+  description: Issues and PRs related to the python.d data collection framework
+  color: *collectors
+- name: collectors/slabinfo
+  description: Issues and PRs related to the slabinfo data collection plugin
+  color: *collectors
+- name: collectors/statsd
+  description: Issues and PRs related to the statsd data collection framework
+  color: *collectors
+- name: collectors/tc
+  description: Issues and PRs related to the tc data collection plugin
+  color: *collectors
+- name: collectors/timex
+  description: Issues and PRs related to the timex data collection plugin
+  color: *collectors
+- name: collectors/xenstat
+  description: Issues and PRs related to the xenstat data collection plugin
+  color: *collectors
+
+# CI Override Labels
+#
+# These are interpreted by our CI when seen on PRs
+# Usually, they force running some CI that would not normally run on that PR
+- name: run-ci/actionlint
+  description: Force running actionlint against labeled PRs
+  color: &run-ci b1a6fb
```

Comment:
This is the unique color defined with `&run-ci`, is this a typo error?

Reply:
Nope, it’s defining a YAML anchor with that exact value, which is then referenced in every other `run-ci` label’s color definition.